src/DGtal/shapes/doc/moduleSurfaceMesh.dox

```diff
@@ -235,5 +235,76 @@ corresponding to relative distances 0.25, 0.5 and 0.75.
 
 @image html spot-bft-isolines.png "SurfaceMesh faces colored according to distance to bluest vertex and three isodistance lines"
 
+\section Flipping edges
+
+Since 1.4, you may modify a SurfaceMesh through flips. Only edges that
+are incident to a triangle on the right and a triangle on the left are
+(topologically) flippable. These two triangles form then a
+quadrilateral, and the edge is one diagonal of this
+quadrilateral. Flipping the edge means that the quadrilateral is now
+split only the other diagonal. Flipping twice the same edge restores
+the original mesh.
+
+The following methods are useful for flipping an edge:
+- SurfaceMesh::isFlippable indicates if a given edge is flippable.
```

Comment:
Maybe you could refer to the API doc or add details here (e.g. "(on triangular meshes, the edge has exactly two adjacent faces...)")

Reply:
Done